Build/SaveLoadScreen.cc

```diff
@@ -994,7 +1219,7 @@ static void SelectedSaveRegionCallBack(MOUSE_REGION* pRegion, INT32 iReason)
 */
 
 		//If we are saving and this is the quick save slot
-		if( gfSaveGame && bSelected == 0 )
+		if( gfSaveGame && bSelected == 0 && gfActiveTab == 0)
```

Comment:
That would actually be wrong here, since this is a check for the QuickSave slot which can only appear on the first page. So a check for the first page is more appropriate.